libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Directory.Build.props

```diff
@@ -0,0 +1,28 @@
+<!-- Contains common properties that apply to projects under the Libraries folder -->
+<Project>
+  <PropertyGroup>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GeneratePackageOnBuild>$(GeneratePackages)</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
We use it in all our libraries. More info on source link [here](https://github.com/dotnet/sourcelink). It gets used in all the projects in the `libraries` folder because of the scoped [Directory.Build.Props](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Directory.Build.props).